attestation-service/verifier/src/snp/mod.rs

```diff
@@ -253,16 +286,25 @@ pub(crate) fn parse_tee_evidence(report: &AttestationReport) -> TeeEvidenceParse
     claims_map as TeeEvidenceParsedClaim
 }
 
+fn get_common_name(cert: &x509::X509) -> Result<String> {
+    let mut entries = cert.subject_name().entries_by_nid(Nid::COMMONNAME);
+    let Some(e) = entries.next() else {
+        bail!("No CN found");
+    };
```

Comment:
I noticed that in the previous version there is a sentence
```rust
assert_eq!(entries.count(), 0);
```
when `entries.next()` is `Some`

Just please double check if it is intended to delete the condition judgement. If else, please use
```rust
if e.count() != 0 {
    bail!(...)
}
```